tests/test_complete_rules.py

```diff
@@ -289,6 +290,43 @@ def match(self, node):
         return AsyncifierAction(node)
 
 
+class OnlyKeywordArgumentDefaultNotSetCheckRule(refactor.Rule):
+    context_providers = (context.Scope,)
+
+    INPUT_SOURCE = """
+        class Klass:
+            def method(self, *, a):
+                print()
+                
+            lambda self, *, a: print
+
+        """
+
+    EXPECTED_SOURCE = """
+        class Klass:
+            def method(self, *, a):
+                print()
+                
+            lambda self, *, a: print
+
+        """
+
+    def match(self, node: ast.AST):  # type: ignore
```

Comment:
Instead of having a `#type: ignore` here, would you mind returning a real action that does a rewrite (and transforms the initial `INPUT_SOURCE` into `EXPECTED_SOURCE`). For example a `ReplacementAction` where you rewrite the function signature to get rid of asterisk.